index.js

```diff
@@ -616,6 +616,9 @@ function speedTest(options) {
   nextServer();
 
   function gotServers(err, servers) {
+    if (!servers.hasOwnProperty("settings") || !servers.settings.hasOwnProperty("servers") || servers.settings.servers.length == 0) {
+      err = "No server found, verify your proxy/network";
```

Comment:
This should be an actual error object. Also, the convention is to use single quote strings.

```
err = new Error('No server found, verify your proxy/network');
```